src/vector.jl

```diff
@@ -55,11 +54,12 @@ function Base.copy!(
             throw(BoundsError(src, soff:soff+n-1))
         end
     end
-    if doff < soff
-        @inbounds for i in 0:n-1 dest[doff+i] = src[soff+i] end
-    else
-        @inbounds for i in n-1:-1:0 dest[doff+i] = src[soff+i] end
-    end
+    # Use memmove for speedy copying. Note: this correctly handles overlapping regions.
+    blob_size = Blobs.self_size(T)
+    ccall(:memmove, Ptr{Cvoid}, (Ptr{Cvoid}, Ptr{Cvoid}, Csize_t),
```

Comment:
Small style comment: according to RAIStyle (https://github.com/RelationalAI/RAIStyle#method-definitions), method calls and definitions that exceed the line limit, should be formatted like so

```
foo(
    long_argument_name_1,
    long_argument_name_2,
    long_argument_name_3,
    etc
)
````

Reply:
Thanks, done.